app/views/shared/_header_navbar.html.erb

```diff
@@ -1,5 +1,5 @@
 <header class="lux">
-  <banner>
+  <banner dismissible>
```

Comment:
I think it would be good to have this change in a separate PR

Reply:
Sounds good, will do!

Reply:
Sorry, @taylor-yamashita ! That was my fault. You asked me, and I thought it was a small enough change that you could add it here, but I was wrong.